assets/README.md

```diff
@@ -58,6 +58,22 @@ loaded in the first place. Typical usage: `assetManager.unloadSafely("test.png")
 exception thrown during reloading. Note that `AssetManager` can throw `GdxRuntimeException` if the asset was not loaded yet.
 - `AssetManager.getLoader` and `setLoader` extension methods with reified types added to ease handling of `AssetLoader`
 instances registered in the `AssetManager`.
+- The `AssetGroup` class is provided for easily grouping together assets so they can be managed as a group through calls 
+such as `loadAll()` or `unloadAll()`. The intended use is to subclass `AssetGroup` and 
+list its member assets as properties using `AssetGroup.asset()` or `AssetGroup.delayedAsset()`. It also allows for using 
+a common prefix for the file names of the group in case they are stored in a specific subdirectory. For example:
+```Kotlin
+class MapScreenAssets(manager: AssetManager) : AssetGroup(manager, "mapScreen/"){
+    val atlas by asset<TextureAtlas>("atlas.json")
+    val music by asset<Music>("mapScreen.ogg")
+}
+
+val mapScreenAssets = MapScreenAssets(manager)
+//...
+manager.finishLoading()
```

Comment:
README examples should generally showcase the most common use case. I'd argue that `finishLoading` would be rarely used in practice. Could you replace it with `loadAll`/`update` loop or something similar?

Reply:
Two separate examples for `delayedAsset`/`asset` showcasing would also be nice.

Reply:
I left a comment explaining how you'd use it differently. Keep it concise, or add a second example class to expand on the differences?

Reply:
I think 2 separate examples is the way to go, since there seem to be 2 approaches to managing assets with groups and joining can be tricky. You can move them to the `Usage examples` section.

Reply:
I expanded it in a concise way. Seems OK to keep in this section.